baseframe/filters.py

```diff
@@ -211,3 +212,39 @@ def longdate(value):
     else:
         dt = value
     return dt.strftime('%e %B %Y')
+
+
+@baseframe.app_template_filter('date')
+def date_filter(value, format='medium'):
+    if isinstance(value, datetime):
+        if value.tzinfo is None:
+            dt = UTC.localize(value).astimezone(get_timezone())
+        else:
+            dt = value.astimezone(get_timezone())
+    else:
+        dt = value
+    return format_date(dt, format=format, locale=get_locale())  # NOQA
```

Comment:
Use the specific `NOQA` code. For example, if it's an error `E001`, use `# noqa: E001`. This will prevent ignoring _other_ issues.